guide/principles/principles.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: guide
+title: Principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 2
+has_children: false
+permalink: /guide/principles/principles/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+# Bitcoin Design Principles
+
+As a new technology, Bitcoin offers the opportunity of an open financial system without intermediaries. To make this a reality, we encourage everyone that is working on products to deliberately support the core principles of designing for Bitcoin. Some of these come from the technology itself, and others from the behaviour and ethos of the community. Although every use case and product is different, all applications should strive to follow these principles. Diverging from them should only be done with very good reason.
```

Comment:
> As a new technology, Bitcoin offers the opportunity of an open financial system without intermediaries. To make this a reality, we encourage everyone that is working on products to deliberately support the core principles of designing for Bitcoin.

The intro sounds more related to "Permissionless" than "Decentralization", perhaps another principles, is it worth distinguishing the two?

Reply:
Good point. I'd like to combine the two if possible.
I see the permission-less aspect as a result of the decentralization, but perhaps it can be expressed better.
Will add decentralized in the first sentence for now.

Reply:
I also think why `an open financial system without intermediaries` is important is not immediately obvious, maybe we just say it's available for everyone but link the trust minimized nature to decentralization

Reply:
Addressed in 3b98813